vignettes/compareToBaseR.Rmd

```diff
@@ -0,0 +1,1202 @@
+---
+title: "Comparing stringr to base R"
+author: "Sara Stoudt"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+This vignette aims to compare stringr functions to base R equivalents. Following the groupings of functions from the [stringr cheat sheet](https://github.com/rstudio/cheatsheets/blob/master/strings.pdf), these are the base equivalents to stringr functions.
+
+# Detect Matches
+
+| base | stringr    |
+|------|------------|
+|   `grepl()`   | `str_detect()` |
+|   `grep()`   | `str_which()`  |
+|   `gregexpr()`   | `str_count()`  |
+|  `gregexpr()`    | `str_locate()` |
+
+# Subset Strings
+
+| base | stringr    |
+|------|------------|
+|   `substr()`   | `str_sub()`    |
+|    `grep()`  | `str_subset()` |
```

Comment:
I'd use `grep(value = TRUE)` just to reinforce that this is a specific use of `grep()`

Reply:
done in commit 70d0f45